cmd/runtimetest/main.go

```diff
@@ -342,6 +344,71 @@ func validateRootFS(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateRootfsPropagation(spec *rspec.Spec) error {
+	if spec.Linux == nil {
+		return nil
+	}
+
+	targetDir, err := ioutil.TempDir("/", "target")
```

Comment:
Can you change the create path? Perhaps `ioutil.TempDir("", "target")`? If you do not change, `runtimetest` must use `sudo`.